components/about/contactForm.js

```diff
@@ -1,22 +1,97 @@
-import {
-  Box,
-  Text,
-  Link
-} from '@chakra-ui/react'
+/* global fetch */
+import { useState } from 'react'
+import { Input, Textarea, Button, FormControl, FormLabel, FormHelperText } from '@chakra-ui/react'
 
 const ContactForm = () => {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [subject, setSubject] = useState('')
+  const [message, setMessage] = useState('')
+
+  const submitHandler = (e) => {
+    e.preventDefault()
+
+    const data = {
+      name,
+      email,
+      subject,
+      message
+    }
+
+    fetch('/api/contact', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json, text/plain, */*',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(data)
+    }).then((res) => {
+      if (res.status === 200) {
+        setName('')
+        setEmail('')
+        setSubject('')
+        setMessage('')
+      } else {
+      }
+    }).catch((err) => {
+      // TODO: Add error handling
+      console.log('ERROR:', err)
+    })
```

Comment:
Perhaps if sending the message fails it should notify the user in some way so they aren't expecting that it worked behind the scenes. Can the UI be updated with some kind of failure message/toast ?

Reply:
Agreed. Added Charka Toast to notify user email was sent or if there is an error. Still needs better error handling, because it's just a catchall right now, but it works fine for now.